docs/framework/install/on-windows-7.md

```diff
@@ -18,13 +18,13 @@ The .NET Framework is required to run many applications on Windows. You can use
 
 ![This application could not be started](./media/this-application-could-not-be-started.png)
 
-These instructions will help you install the .NET Framework versions you need. The [.NET Framework 4.7.1](https://www.microsoft.com/en-us/download/details.aspx?id=56115&desc=dotnet47) is the latest version. It is supported on Windows 7 and Windows Server 2008 R2 and is included with the [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10) and [Windows Server 2016 Version 1709](https://docs.microsoft.com/windows-server/get-started/get-started-with-1709).
+These instructions will help you install the .NET Framework versions you need. The [.NET Framework 4.7.2](http://go.microsoft.com/fwlink/?LinkID=863255) is the latest version. It is supported on Windows 7 SP1 and Windows Server 2008 R2 and is included with [Windows 10 RS4](https://www.microsoft.com/software-download/windows10).
 
-## .NET Framework 4.7.1
+## .NET Framework 4.7.2
 
-* [Download the .NET Framework 4.7.1](https://www.microsoft.com/net/framework/versions/net471?utm_source=ms-docs&utm_medium=referral)
+* [Download the .NET Framework 4.7.2](https://www.microsoft.com/net/framework/versions/net472?utm_source=ms-docs&utm_medium=referral)
```

Comment:
wondering if we should change this here and in the other topics to follow the button I did for the Win 10 version. I did that to test if people would stop complaining about not finding the download (it also takes you straight to the download, instead a middle page)